eng/pipelines/templates/jobs/tests-nightly-python.yml

```diff
@@ -46,5 +46,5 @@ jobs:
         condition: always()
         inputs:
           testResultsFiles: '**/junit/test-results.xml'
-          testRunTitle: '$(OSName) Python $(PythonVersion)'
+          testRunTitle: '$(OSVmName) Python $(PythonVersion)'
```

Comment:
You should leave this as $(OSName) since it will be set by the verify-agent-os.yml script [here](https://github.com/Azure/azure-sdk-for-python/blob/9d242098da666311c76ae82de07bf5161c7929dd/eng/common/pipelines/templates/steps/verify-agent-os.yml#L36)